pandas/tests/io/parser/dtypes/test_dtypes_basic.py

```diff
@@ -385,3 +385,45 @@ def test_dtypes_defaultdict_invalid(all_parsers):
     parser = all_parsers
     with pytest.raises(TypeError, match="not understood"):
         parser.read_csv(StringIO(data), dtype=dtype)
+
+
+def test_use_nullabla_dtypes(all_parsers):
+    # GH#36712
+
+    parser = all_parsers
+
+    data = """a,b,c,d,e,f,g,h,i
+1,2.5,True,a,,,,,12-31-2019
```

Comment:
Could you add a column here where both rows have an empty value?

Reply:
Added, casts to Int64 now in both cases. Better question is what we actually want here, because this could be everything